File: tests/fixtures/runnable_julia_minimal/code/tables.jl
using Statistics

x = collect(1.0:20.0)

open("tables/summary.tex", "w") do io
    println(io, "\\begin{tabular}{lr}")
    println(io, "\\toprule")
    println(io, "Statistic & Value \\\\")
    println(io, "\\midrule")
    println(io, "Mean & $(round(mean(x); digits=2)) \\\\")
    println(io, "Std & $(round(std(x); digits=2)) \\\\")
    println(io, "\\bottomrule")
    println(io, "\\end{tabular}")
end

File: tests/fixtures/runnable_python_minimal/code/tables.py
import statistics

x = list(range(1, 21))

with open("tables/summary.tex", "w") as f:
    f.write("\\begin{tabular}{lr}\n")
    f.write("\\toprule\n")
    f.write("Statistic & Value \\\\\n")
    f.write("\\midrule\n")
    f.write(f"Mean & {statistics.mean(x):.2f} \\\\\n")
    f.write(f"Std & {statistics.stdev(x):.2f} \\\\\n")
    f.write("\\bottomrule\n")
    f.write("\\end{tabular}\n")
